packages/argo-run/src/dev.ts

```diff
@@ -185,10 +185,10 @@ export async function dev(...args: string[]) {
             {
               scriptUrl: `${origin}${PUBLIC_PATH}${filename}`,
               socketUrl: `${origin.replace(/^http/, 'ws')}${WEBSOCKET_PATH}`,
-              extensionPoint:
+              extensionPoints:
                 extension.type === 'checkout'
-                  ? extension.config.extensionPoints[0]
-                  : 'Checkout::PostPurchase::Render',
+                  ? extension.config.extensionPoints
+                  : ['Checkout::PostPurchase::Render'],
```

Comment:
Considering this `/query` endpoint is not being used for PostPurchase, I think it's safe to change to `extensionPoints`.